vortexasdk/client.py

```diff
@@ -195,6 +197,23 @@ def create_client() -> VortexaClient:
         raise KeyError(
             "VORTEXA_API_KEY environment variable is required to use the VortexaSDK"
         )
+    latest_version, _ = get_latest_package_version(pkg_name)
+    try:
+        version_mask = StrictVersion(__version__) < StrictVersion(
+            latest_version
+        )
+    except ValueError as e:
+        if "invalid version number" in str(e):
+            version_mask = LooseVersion(__version__) < LooseVersion(
+                latest_version
+            )
+        else:
+            raise e
+    if version_mask:
+        logger.warning(
+            f"You are using vortexasdk version {__version__}, however version {latest_version} is available.\n"
+            f"You should consider upgrading via the 'pip install vortexasdk --upgrade' command."
+        )
```

Comment:
I think this is a good place to put the logic 👍 .

Can we extract this into a separate function though, then just call it in a single line in the `create_client` method

Reply:
Any yes, that's a good idea. The entire content of the `warn_user_if_sdk_is_outdated` method can be wrapped in a try-catch method just in case it blows up, so as not to prevent the client from being used.

Reply:
All of those are fair points. Will apply the requested changes and revert.